### PR TITLE
fix: fixes lint issues and generates metadata

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@g-awmalik
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@g-awmalik
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: autogen/versions.tf.tmpl [hunk 1]
```diff
@@ -26,6 +26,10 @@ terraform {
       source  = "hashicorp/google-beta"
       version = ">= 4.40, < 5.0"
     }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.4"
```
Comment (apeabody) [inline autogen/versions.tf.tmpl:31]: Let's relax this constraint if possible to reduce the likelyhood of a breaking change.  This is what we have been using for other modules that didn't previously include a version for random:

```suggestion
      version = ">= 2.1"
```

File: modules/dynamic_backends/versions.tf [hunk 2]
```diff
@@ -26,6 +26,10 @@ terraform {
       source  = "hashicorp/google-beta"
       version = ">= 4.40, < 5.0"
     }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.4"
```
Comment (apeabody) [inline modules/dynamic_backends/versions.tf:31]:
```suggestion
      version = ">= 2.1"
```

File: versions.tf [hunk 3]
```diff
@@ -26,6 +26,10 @@ terraform {
       source  = "hashicorp/google-beta"
       version = ">= 4.40, < 5.0"
     }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.4"
```
Comment (apeabody) [inline versions.tf:31]:
```suggestion
      version = ">= 2.1"
```

File: modules/serverless_negs/versions.tf [hunk 4]
```diff
@@ -26,6 +26,10 @@ terraform {
       source  = "hashicorp/google-beta"
       version = ">= 4.40, < 5.0"
     }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.4"
```
Comment (apeabody) [inline modules/serverless_negs/versions.tf:31]:
```suggestion
      version = ">= 2.1"
```

File: autogen/versions.tf.tmpl (hunk 1 above)
Comment (g-awmalik) [reply autogen/versions.tf.tmpl:31]: done

**Comment by comment-bot-dev** [comment]
@g-awmalik
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@g-awmalik
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by apeabody — APPROVED** [review]
Thanks @g-awmalik!

**Pull request merged**